src/main/java/io/jenkins/plugins/tuleap_oauth/checks/UserInfoCheckerImpl.java

```diff
@@ -15,6 +15,10 @@ public boolean checkUserInfoResponseBody(UserInfo userInfo, DecodedJWT idToken)
             LOGGER.warning("Subject not expected");
             return false;
         }
+        if (!userInfo.isEmailVerified()) {
+            LOGGER.warning("Invalid user");
+            return false;
```

Comment:
do we really want to lock out people without verified mail ?

Reply:
Yes because i do not think we want an user who is either anonymous, or deleted, or suspended can be connected.

According to the Tuleap implementation, an email is verified if the user is not anonymous and if he is active or restricted